frontend/src/views/accessibilityanalysis.js

```diff
@@ -465,44 +463,15 @@ export class AccessibilityAnalysisView extends React.Component {
                     width={300}
                   />
               </div>
-
-              }
-              {false &&
```

Comment:
@jjaquish same as above, why the `bool && <element>` here - in this case with a true and a false?

Reply:
Eliminate it!